### DISCUSSION TITLE
Codespace option not available for github classroom.

### DISCUSSION THREAD

**Discussion by woensug-choi** [community/community#38740] [GitHub Education]
I've followed through the document about using codespace in GitHub classroom(https://docs.github.com/en/education/manage-coursework-with-github-classroom/integrate-github-classroom-with-an-ide/using-github-codespaces-with-github-classroom).

But I don't codespace icon on the organization selection page when creating a new classroom.

I've upgraded to a Teams at Github education. And added payment methods and set the pay limit for codespace in the organization settings. 
Yet, newly created classroom settings still show the 'verify' button for codespace (which leads to a faculty member verification page saying that I've already been verified). 

Please help.

**Comment by ossanna16** [comment]
Hi there @woensug-choi and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
